download_and_create_reference_datasets/v03/update_combined_interval_reference_ht.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
```

Comment:
My first pass here tried to preserve keeping `combined_interval_reference` and `combined_reference` in separate scripts.  I think those scripts are gonna end up looking basically identical though.